src/main/java/frc/robot/subsystems/Rollergripper.java

```diff
@@ -131,24 +128,16 @@ public Command manualCommand(Trigger intake, Trigger outtake) {
           () -> {
             if (intake.getAsBoolean() || outtake.getAsBoolean()) {
               if (intake.getAsBoolean()) {
-                rightRoller.set(0.6);
-                leftRoller.set(0.6);
+                LeadRoller.set(0.6);
```

Comment:
same

Reply:
what's going on, I don't see it in my code